teams/wg-prioritization.toml

```diff
@@ -3,10 +3,9 @@ subteam-of = "compiler"
 kind = "working-group"
 
 [people]
-leads = ["spastorino", "wesleywiser"]
+leads = ["apiraino", "spastorino", "wesleywiser"]
 members = [
     "am-1t",
-    "apiraino",
```

Comment:
We don't have to remove them from the member list.

Reply:
In fact, you *can't*:

```
ERROR validation error: `apiraino` leads team `wg-prioritization`, but is not a member of it
ERROR validation error: person `apiraino` is a member of a mailing list but has no email address
ERROR 2 validation errors found
```

Reply:
@camelid I've noticed the second validation warning ("... is a member of a mailing list but has no email address"). Should I add my public email somewhere? (no idea if it's needed for something)

Reply:
Yes, you should add it to `people/apiraino.toml`.

Reply:
Added Antonio's email.

Reply:
This removed line should be re-added.